racetime/views/race.py

```diff
@@ -97,6 +97,10 @@ class RaceMini(Race):
     template_name_suffix = '_mini'
 
 
+class RaceLivesplit(Race):
+    template_name_suffix = '_livesplit'
```

Comment:
I think we should probably restrict this view to authenticated users only. I don't see any need for it if you're logged out, and it's not something I want Google to stumble on.